docs/elasticsearch/options.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Common Options
+parent: Elasticsearch
+nav_order: 100
+---
+
+# Common options
+
+The following options are available for all REST operations:
+
+Option | Valid values
+:--- | :--- | :---
+Time unit | The supported units for time are `d` for days, `h` for hours, `m` for minutes, `s` for seconds, `ms` for milliseconds, `micros` for microseconds, and `nanos` for nanoseconds.
```

Comment:
Because this will be on a dedicated Units page, we can just say "Times," "Bytes," "Distances," and "Quantities without units." (I think "unit-less quantities" is hard to understand.)